test_runner/regress/test_tenant_relocation.py

```diff
@@ -40,58 +34,6 @@ def assert_abs_margin_ratio(a: float, b: float, margin_ratio: float):
     assert abs(a - b) / a < margin_ratio, abs(a - b) / a
 
 
-@contextmanager
-def new_pageserver_service(
```

Comment:
just to make sure: this is not a test but it is a helper function that is now redundant? python syntax is a little implicit for me to figure it out for sure just from looking at the function definition.

Reply:
Right.  This test used to have some special code for constructing additional pageserver instances, but since NeonEnv got the ability to run multiple pageservers, it is no longer needed.  I cleaned it up in this PR so that the test would have accesss to proper NeonPageserver objects, as their generation-aware helpers are now needed.